controller/config.yaml

```diff
@@ -0,0 +1,17 @@
+processor:
+  ignoreTypes:
+    - "(Elasticsearch|Kibana|ApmServer)List$"
```

Comment:
These specific ignore types aren't useful for us I think, we should probably do something like `.*List$` though